### PR TITLE
GIBS Feedback Tool Landing Page Content edits

### PR DESCRIPTION
**Pull request by cehsu**
localhost_3001_education_complaint-tool_form_introduction 4

### CODE REVIEW

File: src/applications/edu-benefits/complaint-tool/containers/IntroductionPage.jsx [hunk 1]
```diff
@@ -15,19 +15,19 @@ class IntroductionPage extends React.Component {
     return (
       <div className="schemaform-intro">
         <FormTitle title="GI Bill® School Feedback Tool"/>
-        <p>You can submit feedback to us if your GI Bill school isn’t following the Principles of Excellence guidelines or if you have any other concerns or issues you’d like to raise with us. The Principles of Excellence program requires schools to follow certain guidelines in order to get federal funding through the GI Bill.</p>
-        <p>You can choose to submit feedback anonymously or on behalf of someone else. Any feedback sent in anonymously isn’t shared with schools or employers.</p>
+        <p>If you have an issue or complaint about a school or training facility that receives GI Bill benefits, you can submit feedback to VA.</p>       <p>You can choose to submit feedback anonymously or on behalf of someone else. We’ll share any information you provide with the schools. If you submit feedback on behalf of someone, we’ll share only your information with the school, and not the person you’re submitting on behalf of.</p>
```
Comment (erikphansen) [inline src/applications/edu-benefits/complaint-tool/containers/IntroductionPage.jsx]: nit: can these two `p` tags be on different lines?

File: src/applications/edu-benefits/complaint-tool/containers/IntroductionPage.jsx [hunk 2]
```diff
@@ -56,6 +58,9 @@ class IntroductionPage extends React.Component {
           startText="Submit Your Feedback"
           {...this.props.saveInProgressActions}
           {...this.props.saveInProgress}/>
+        <div className="omb-info--container" style={{ paddingLeft: '0px' }}>
```
Comment (erikphansen) [inline src/applications/edu-benefits/complaint-tool/containers/IntroductionPage.jsx]: do we have to do this padding override a lot?

File: src/applications/edu-benefits/complaint-tool/containers/IntroductionPage.jsx [hunk 3]
```diff
@@ -36,16 +37,17 @@ class IntroductionPage extends React.Component {
...
             <li className="process-step list-four">
-              <div><h5>VA Follow Up</h5></div>
-              <p>We’ll get back to you within 45 days to let you know how we’re handling your feedback and if we’ve had any communication with your school. Feedback that isn’t related to VA education benefits may be sent to another agency for review.</p>
+              <div><h5>School Follow Up</h5></div>
+              <p>The school will review your feedback and send us their response.</p>
             </li>
             <li className="process-step list-five">
-              <div><h5>School Follow Up</h5></div>
-              <p>The school will review your feedback. We’ll send you the school’s response and ask you if you think their response resolves your issue. In some cases, the school may get in touch with you directly to discuss your feedback.</p>
+              <div><h5>VA Follow Up</h5></div>
```
Comment (erikphansen) [inline src/applications/edu-benefits/complaint-tool/containers/IntroductionPage.jsx]: I need to check to see if we actually need to wrap these `h5` tags in `div` tags

**Review by erikphansen — APPROVED** [review]
Just a nit and one question.

**Review by rianfowler — APPROVED** [review]
Ticket says it should be School Review instead of School Follow Up

File: src/applications/edu-benefits/complaint-tool/containers/IntroductionPage.jsx (hunk 2 above)
Comment (cehsu) [reply src/applications/edu-benefits/complaint-tool/containers/IntroductionPage.jsx]: We should create a TODO item to create a class for this, good point. I will create that in a subsequent PR.

File: src/applications/edu-benefits/complaint-tool/containers/IntroductionPage.jsx (hunk 3 above)
Comment (cehsu) [reply src/applications/edu-benefits/complaint-tool/containers/IntroductionPage.jsx]: Good question, let's revisit this. I will set up a todo to check this in a subsequent PR.

**Comment by peggygannon** [comment]
@cehsu Based on today's stakeholder meeting, we have some additional edits to this page (sorry!) Please see below. Let me know if you have any questions: 

New intro: 
If you have an issue or complaint about a school or training facility that’s eligible to receive GI Bill benefits, you can submit feedback to VA. You can submit feedback to us if your school isn’t following the Principles of Excellence guidelines or if you have any other concerns or issues you’d like to raise with us.

You can choose to submit your feedback anonymously or on behalf of someone else. We share all information with the school, but if you submit anonymously we won’t share your name with the school. 
Get started right now by filling out a short form. Follow the steps below. 

Under **Prepare** header
Please update first bullet/line: Provide your school’s name and address
In 3rd bullet, delete comma before “so”
please add a period to the end of the last bullet

Under **VA Review** update content to:
We’ll review your feedback and pass it along to your school for their review. Feedback that isn’t related to VA education benefits may be sent to another agency for review. 
We’ll get back to you within 45 days to let you know how we’re handling your feedback. 

Delete **School Follow Up** bullet

Change **VA Follow Up** to **VA Processing** and update content to:
After we get the school’s response to your feedback, we’ll send it to you and ask if you think their response resolves your issue. In some cases the school may also get in touch with you directly about your feedback.

**Pull request merged**